andela/wg-a-team
# Ft upgrade nutrition

**What this PR does**
Allows User to add meals they consumed during the day
**Description of Task to be completed?**
A user should have a field where they enter the food they consumed that day
**How should this be manually tested?** 
Navigate to a single nutrition plan and press the Edit icon. Among other fields, a meal consumed field should appear where text(Meals consumed) can be entered.
**What are the relevant pivotal tracker stories?**
#152919325
